tests/unit/test_mxnet.py

```diff
@@ -304,7 +348,8 @@ def test_mxnet(strftime, sagemaker_session, mxnet_version, skip_if_mms_version):
 
     model = mx.create_model()
 
-    expected_image_base = "520713654638.dkr.ecr.us-west-2.amazonaws.com/sagemaker-mxnet:{}-gpu-py2"
+    # since not mms, going to be old images always, so hard-coded
```

Comment:
I would explain here that it's because the `mxnet_version` fixture only contains pre-MMS versions

Reply:
will do